src/plugins/ncr/components/labels-picker/index.jsx

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import chroma from 'chroma-js';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import isEqual from 'lodash/isEqual';
```

Comment:
@rsumilang this should probably be called labels-form since it's nto really a picker, it just happens to have one